docs/source/install/upgrades.rst

```diff
@@ -28,10 +28,7 @@ Upgrade Procedure
 
    sudo st2ctl stop
 
-2. Run the migration script (if any). See section below for |st2|
```

Comment:
@dzimine  Thanks for looking. I have changed the sequence and havn't removed step for running migration script. Basically, the migration scripts can be run once the upgrade of packages (eg. st2) has been done. The doc currently asks user to run the migration script first and then do the package upgrade which does not work.

@Kami  Could you please have a look at this PR and let me know if the changes are OK or not.